books/free-programming-books.md

```diff
@@ -467,6 +468,17 @@
 * [Theory and Applications for Advanced Text Mining](http://www.intechopen.com/books/theory-and-applications-for-advanced-text-mining)
 
 
+#### Game Development
+
+* [2D Game Development: From Zero To Hero](https://github.com/Penaz91/2DGD_F0TH) (:construction: *in process*)
```

Comment:
Maybe better use deployed url seen in repo?

https://therealpenaz91.itch.io/2dgd-f0th

Reply:
I think it's not necessary, in the repository there are some information that there aren't in the other site. I let the staff decide what to do.

Reply:
Ok. Let @Penaz91 decide and if needed threat in other issue.

Reply:
The ebook has two main repositories where the sources can be found (both on Gitlab and Github) but the epub and pdf versions are published on itch just for ease of deployment. Feel free to reference any url you wish.